content/en/cosign/signing_with_containers.md

```diff
@@ -20,38 +29,28 @@ This usage is a common use case that uses traditional key signing from a key pai
 $ cosign sign --key cosign.key user/demo
 ```
 
-If you need to generate local keys, you can do so by running `cosign generate-key-pair`.
-
-## Keyless signing
-
-You can use Cosign to sign with ephemeral keys by authenticating with an OIDC (OpenID Connect) protocol supported by Sigstore. Currently, you can authenticate with Google, GitHub, or Microsoft. For more information, read [Keyless Signatures](/cosign/keyless/).
+If you need to generate local keys, you can do so by running `cosign generate-key-pair`. 
```

Comment:
Same comment here, should we link to the page on signing with self-managed keys?